src/index.js

```diff
@@ -3,10 +3,14 @@ import habitat from "preact-habitat";
 
 import Widget from "./containers/widget";
 
+import InlineWidget from "./containers/inline-widget";
+
 let _habitat = habitat(Widget);
 
 _habitat.render({
   selector: ".__fourohtwo",
   clean: true,
   component: "area402",
 });
+
```

Comment:
why not something like: 

```js
let _habitatWidget = habitat(Widget);
let _habitatWidget = habitat(InlineWidget);

_habitatWidget.render({
  selector: "SOME WIDGET SPECIFIC SELECTOR",
  clean: true,
  component: "area402",
});

_habitatWidget.render({
  selector: ".SOME INLINE SPECIFIC SELECTOR",
  clean: true,
  component: "area402Inline",
});

```

Reply:
I think this has the advantage that both exactly work the same and it is consistent.

Reply:
Agreed, this has an advantage. Used this approach and it works.